programs/ziti-edge-tunnel/config-utils.c

```diff
@@ -0,0 +1,64 @@
+/*
+ Copyright 2019-2021 NetFoundry Inc.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ https://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+const char* app_data = "APPDATA";
+static char* identifier_path = NULL;
+
+char* get_system_config_path() {
+    char* config_path = malloc(FILENAME_MAX * sizeof(char));
+#if _WIN32
+    sprintf(config_path, "%s/NetFoundry", getenv(app_data));
+#else
+    sprintf(config_path, "%s", "/tmp");
```

Comment:
this is bad place to store config: `/tmp` is wiped on reboot

Reply:
I have moved config-utils.c to Windows in CmakeLists, I will fix this else part anyway.

Reply:
fixed the config path for __linux__ env. When we use this param for other platforms, we will fix it for other platforms. Right now, it is being used for windows only